Source/Core/DolphinQt/Config/ConfigControls/ConfigFloatSlider.h

```diff
@@ -0,0 +1,30 @@
+// Copyright 2017 Dolphin Emulator Project
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#include "DolphinQt/Config/ToolTipControls/ToolTipSlider.h"
+
+namespace Config
+{
+template <typename T>
+class Info;
+}
+
+// Automatically converts an int slider into a float one.
+// Do not read the int values or ranges directly from it.
+class ConfigFloatSlider : public ToolTipSlider
```

Comment:
class `ConfigFloatSlider final`

Reply:
The other Config GUI classes aren't final either, though you're right that they probably could be. Maybe open up a separate PR for that?